addons/hr_timesheet/wizard/project_task_create_timesheet.py

```diff
@@ -20,13 +20,17 @@ class ProjectTaskCreateTimesheet(models.TransientModel):
     )
 
     def save_timesheet(self):
+        minimum_duration = int(self.env['ir.config_parameter'].sudo().get_param('hr_timesheet.timesheet_min_duration', 0))
+        rounding = int(self.env['ir.config_parameter'].sudo().get_param('hr_timesheet.timesheet_rounding', 0))
+        minutes_spent = self.task_id._timer_rounding(self.time_spent * 60, minimum_duration, rounding)
```

Comment:
I guess you did not test your dev 👼🏼 😘 
This cannot work in community without enterprise 🙂 
The reason why it does not break is that there is no test using this code I guess.
You should take the logic from the _timer_rounding  def (coming from the timer mixin) and use it in here directly.

Can you add a test that use this code?

And mercikes 🙂